src/ConnectionConfig.tsx

```diff
@@ -71,6 +71,10 @@ export const ConnectionConfig: FC<ConnectionConfigProps> = (props: ConnectionCon
     loadRegions().then((regions) => setRegions(regions.map(toOption)));
   }, [loadRegions]);
 
+  const inExperimentalAuthComponent = options.jsonData.inExperimentalAuthComponent;
```

Comment:
Just curious why we would want to store this in options.jsonData vs adding this as a prop on `<ConnectionConfig inExperimentalAuthComponent={true} >`

I think the downside to storing this in jsonData is then you need to update all the datasource instances right? Maybe easier to add a temporary flag in the javascript rather than update the user's stored datasource configuration, yeah?

Reply:
@sarahzinger 
Good point! I was having trouble adding the prop to the component with regards to eliminating type errors so I chose to add the flag in the json data, which is not good to update. 

For prometheus, we would have to pass the flag into two components, first the SigV4ConnectionConfig and then again into the ConnectionConfig. I can explore doing that and fixing any typing issues that appear.

Reply:
@sarahzinger Added as a prop now!